ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -448,8 +453,15 @@ class EclCmdPublish : public EclCmdWithEclTarget
         else
             req->setActivate(optNoActivate ? CWUQueryActivationMode_NoActivate : CWUQueryActivationMode_Activate);
 
-        if (optName.length())
-            req->setJobName(optName.get());
+        if (optJobName.length())
+            req->setWorkUnitJobName(optJobName);
+        if (optQueryName.length())
+        {
+            //very confusing parameter name, but set for backward compatability
+            //  for old servers... will not override WorkUnit JobName from call to doDeploy above, so that job name will still win
+            req->setJobName(optQueryName);
```

Comment:
Is this for old server?

Reply:
Yes, setting this keeps old servers working as intended.